pkg/local/local.go

```diff
@@ -18,13 +18,19 @@ type Local struct {
 // CopyConfig takes the name of a service and a Source local service root path to be copied to a Destination.
 //
 // Only files under /path/to/local/repo/config/* are copied to the destination /services/[serviceName]/base/config/*
```

Comment:
The destination is now /environments/env-name/services/[serviceName]/base/config/*